[BUGFIX] Restoration de l'icône svg icon-reload.svg.

jonasgrilleres opened the pull request:
## :unicorn: Problème
Cette icône a été supprimée par mégarde, bien que toujours utilisée.

## 🌈 Remarques 
On en profite pour éditer la config du serveur nginx : si une ressource demandée dans le répertoire `/images` n'est pas trouvée, on renvoie une 404 plutôt que le fichier `index.html`.

## :100: Pour tester
- aller sur une épreuve, 
- faire un signalement sur un problème d'image qui ne s'affiche pas,
- ne pas voir le bug suivant :

![image](https://user-images.githubusercontent.com/5982021/113735378-7db6f780-96fc-11eb-86d8-d89e6bf3393b.png)

jonasgrilleres committed:
Restore deleted svg icon

This icon was deleted by mistake in PR https://github.com/1024pix/pix/pull/2571

pix-service commented:
I'm deploying this PR to these urls:

- App: https://app-pr2816.review.pix.fr
- Orga: https://orga-pr2816.review.pix.fr
- Certif: https://certif-pr2816.review.pix.fr
- Admin: https://admin-pr2816.review.pix.fr
- API: https://api-pr2816.review.pix.fr/api/

 Please check it out!

jbuget reviewed (approved):
Gros boulot ! GG :-)

jonasgrilleres committed:
Add a new nginx rule to allow 404 responses on broken paths in /images

The nginx server is currently configured to fallback on sending the file
index.html when a user requests any not found URL.

There is an exception on the folder /assets. We're adding an other
exception on /images.

jonasgrilleres committed:
[SR] Replace <object> tags with <img> tags

francois2metz reviewed (approved):
:purple_heart: :yellow_heart: :green_heart: :blue_heart: LGTM  :blue_heart: :green_heart: :yellow_heart: :purple_heart:
